spec/controllers/login_controller_spec.rb

```diff
@@ -98,6 +104,12 @@
         expect(flash[:success]).to be_nil
       }.not_to(change { User.find(deleted_wiped.id).password_reset_token })
     end
+
+    it "doesn't show reset form if user is already logged in" do
```

Comment:
Since this is testing forgot password, it would be more appropriate to say something like `doesn't show forgot password form ...`